.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "Docusaurus Dev Container",
+	"image": "mcr.microsoft.com/vscode/devcontainers/typescript-node:0-10-buster",
```

Comment:
how did you choose the image here?

Reply:
We may need something on node LTS 14 instead of node 10

Reply:
Because the GitHub workflow are using nodejs 10 so I chose that

Reply:
The image I chose is one of the official GitHub codespaces images for nodejs with typescript support

Reply:
We may move/require node 14 LTS soon so it's better to use node 14 directly for this feature